include/nuttx/fs/hostfs.h

```diff
@@ -217,7 +226,7 @@ int           host_mkdir(const char *pathname, mode_t mode);
 int           host_rmdir(const char *pathname);
 int           host_rename(const char *oldpath, const char *newpath);
 int           host_stat(const char *path, struct stat *buf);
-
+int           host_stat(const char *path, const struct stat *buf, int flags);
```

Comment:
```suggestion
int           host_chstat(const char *path, const struct stat *buf, int flags);
```
I think there is a mistake here on the function declaration.

Reply:
Yes, done.